umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -46,22 +52,31 @@ If you send the messages into other topics, some features might not work correct
 {{% /notice %}}
 
 ## What are the limitations?
+- Increased complexity in integrating MQTT with Kafka.
+- Deviating from established standards might initially cause discomfort among some users.
+- You have to use the following specific topic structure that adheres to the ISA95 model:
+
+  - **Topic Names and Rules**: All parts of this structure, such as `enterprise`, `site`, `area`, etc., are flexible in terms of their naming. They can include letters (a-z, A-Z), numbers (0-9), and certain symbols (`-` and `_`). However, symbols like `.`, `+`, `#` or `/` are not used as they have special meanings in MQTT or Kafka.
```

Comment:
Can you make it more explicit what symbols are allowed and what not?

Reply:
The link to the data model page is added.